packages/zoe/test/unitTests/zcf/test-zoeHelpersWZcf.js

```diff
@@ -101,6 +104,109 @@ test(`zoeHelper with zcf - swap no match`, async t => {
   assertPayoutAmount(t, moolaIssuer, await bUserSeat.getPayout('A'), moola(5));
 });
 
+test(`zcf usesNatMath`, async t => {
+  const { zcf } = await setupZCFTest();
+  const zcfMint = await zcf.makeZCFMint('A');
+  const { brand } = zcfMint.getIssuerRecord();
+  t.falsy(assertUsesNatMath(zcf, brand), 'default');
+});
+
+test(`zcf usesNatMath - not natMath`, async t => {
+  const { zcf } = await setupZCFTest();
+  const zcfMint = await zcf.makeZCFMint('A', MathKind.SET);
+  const { brand } = zcfMint.getIssuerRecord();
+  t.throws(() => assertUsesNatMath(zcf, brand), {
+    message: 'issuer must use NAT amountMath',
+  });
+});
+
+test(`zcf usesNatMath - not brand`, async t => {
+  const { zcf } = await setupZCFTest();
+  const zcfMint = await zcf.makeZCFMint('A', MathKind.SET);
+  const { issuer } = zcfMint.getIssuerRecord();
+  t.throws(() => assertUsesNatMath(zcf, issuer), {
+    message: '"brand" not found: (an object)\nSee console for error data.',
+  });
+});
+
+test(`zcf usesNatMath - brand not registered`, async t => {
+  const { zcf } = await setupZCFTest();
+  const { brand } = makeIssuerKit('gelt');
+  t.throws(() => assertUsesNatMath(zcf, brand), {
+    message: '"brand" not found: (an object)\nSee console for error data.',
```

Comment:
hmm, we should probably make an issue for making this error different than the error in 128

Reply:
good idea. done: #1800

Reply:
Thanks!!